src/org/ecs160/a2/AppMain.java

```diff
@@ -2,23 +2,26 @@
 
 
 import static com.codename1.ui.CN.*;
-import com.codename1.ui.Display;
-import com.codename1.ui.Form;
-import com.codename1.ui.Dialog;
-import com.codename1.ui.Label;
+
+import com.codename1.components.SpanLabel;
+import com.codename1.ui.*;
+import com.codename1.ui.layouts.BorderLayout;
+import com.codename1.ui.plaf.Border;
+import com.codename1.ui.plaf.Style;
 import com.codename1.ui.plaf.UIManager;
 import com.codename1.ui.util.Resources;
 import com.codename1.io.Log;
-import com.codename1.ui.Toolbar;
+
 import java.io.IOException;
 import com.codename1.ui.layouts.BoxLayout;
 import com.codename1.io.NetworkEvent;
+import java.lang.Object;
 
 /**
  * This file was generated by <a href="https://www.codenameone.com/">Codename One</a> for the purpose 
  * of building native mobile applications using Java.
  */
-public class AppMain {
```

Comment:
Doesn't this class need to be named "AppMain"?

Reply:
That's true! Thanks for catching that.

Reply:
Yeah I was about to say that. Also please remove the comments when you're done.